src/pages/Home/index.tsx

```diff
@@ -66,6 +68,31 @@ const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
 
   // console.log(order, 'this is order3'); // this is for you, u said you want to see the console.log
```

Comment:
Remove this

Reply:
updated at f738575ddca78aaaf7ab71c70f70a0047e756432